arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -163,19 +165,41 @@ public List<ReplayCompareResult> doContentCompare(ReplayActionCaseItem caseItem,
       List<CategoryComparisonHolder> waitCompareMap) {
     ComparisonInterfaceConfig operationConfig = compareConfigService.loadInterfaceConfig(
         caseItem.getParent());
-    List<String> ignoreCategoryList = operationConfig.getIgnoreCategoryTypes();
+    List<CategoryDetail> ignoreCategoryList = operationConfig.getIgnoreCategoryTypes();
 
     List<ReplayCompareResult> replayCompareResults = new ArrayList<>();
     for (CategoryComparisonHolder bindHolder : waitCompareMap) {
       if (operationConfig.checkIgnoreMockMessageType(bindHolder.getCategoryName(),
-          ignoreCategoryList)) {
+          caseItem.getParent().getOperationName(), ignoreCategoryList)) {
```

Comment:
In scenarios where the main interface is not allowed to be ignored, there is no need to pass in operationName.

Reply:
fixed